src/libstd/sys/common/backtrace.rs

```diff
@@ -36,7 +36,7 @@ pub fn log_enabled() -> bool {
     }
 
     let val = match env::var_os("RUST_BACKTRACE") {
-        Some(..) => 2,
+        Some(x) => if x.eq("disabled") { 1 } else { 2 },
```

Comment:
I remembered...

```
sys/common/backtrace.rs:39:23: 39:38 error: the trait `core::cmp::PartialEq<&str>` is not implemented for the type `ffi::os_str::OsString` [E0277]
sys/common/backtrace.rs:39         Some(x) => if x == "disabled" { 1 } else { 2 },
                                                 ^~~~~~~~~~~~~~~
sys/common/backtrace.rs:39:23: 39:38 help: run `rustc --explain E0277` to see a detailed explanation
sys/common/backtrace.rs:39:23: 39:38 help: the following implementations were found:
sys/common/backtrace.rs:39:23: 39:38 help:   <ffi::os_str::OsString as core::cmp::PartialEq>
sys/common/backtrace.rs:39:23: 39:38 help:   <ffi::os_str::OsString as core::cmp::PartialEq<str>>
sys/common/backtrace.rs:39:23: 39:38 help:   <ffi::os_str::OsString as core::cmp::PartialEq<ffi::os_str::OsStr>>
sys/common/backtrace.rs:39:23: 39:38 help:   <ffi::os_str::OsString as core::cmp::PartialEq<&'a ffi::os_str::OsStr>>
sys/common/backtrace.rs:39:23: 39:38 help: and 5 others
error: aborting due to previous error
```

Maybe there's another way to use '==', but I haven't yet reached those tutorials... still in the process of (re)reading rust-book.